### PR TITLE
Attempt to make turtle represent Agent and get a good diagram.

### PR DESCRIPTION
**Pull request by fosterlynn**
This PR is for feedback at this stage.  The picture is getting to where I more or less like it. I don't know if the Turtle is correct, @pmackay 

Note: I created a subClassOf rdf:Property called RelationshipType.  elf had suggested this in a gitter conversation, but we certainly didn't decide it, and elf suggests a lot of things just to brainstorm and get us to think. :)  It does however give us a way to keep all the relationships gathered in one place kind of.  Otherwise all properties will eventually go there. @ahdinosaur what do you think of this (and everything else).

Also, still using vf:label.  And can't remember or find a standard vocab for urls that isn't @id as in json-ld.  This is for the url and the image on Agent.  I'll figure it out, just noting it isn't right.

Since I don't want to merge it, here is what it looks like in vowl:
agent-ttl

### CODE REVIEW

**Comment by pmackay** [comment]
@fosterlynn  If the Turtle is representing the vocab the way you need, its probably fine! I'm learning myself about best practice here.

**Comment by ahdinosaur** [comment]
regarding `image` and `url`, keen to delegate to [as:image](http://www.w3.org/TR/activitystreams-vocabulary/#dfn-image-term) and [as:url](http://www.w3.org/TR/activitystreams-vocabulary/#dfn-url).

File: lynn_temp_agent.ttl [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  a owl:Class ;
+  rdfs:label "Relationship Type" ;
+  rdfs:comment "" ;
+  rdfs:subClassOf rdf:Property .
+
+
+# Properties
+
+vf:label
+  a owl:DatatypeProperty ;
```
Comment (ahdinosaur) [inline lynn_temp_agent.ttl]: do we add  [`owl:equivalentProperty skos:prefLabel`](http://www.w3.org/TR/owl-ref/#equivalentProperty-def) here?

**Comment by ahdinosaur** [comment]
this is looking really great @fosterlynn. :smiley: super keen to be using turtle and vowl.

**Comment by fosterlynn** [comment]
> do we add owl:equivalentProperty skos:prefLabel here?

@ahdinosaur yes, we can add all that extra definitional stuff.  I took some out because it cluttered the diagram too much for my taste.  Maybe 2 versions, one for the diagram and one for the definitions.

And we need to keep in sync with the json-ld context.

> regarding image and url, keen to delegate to as:image and as:url.

Is as the best place to get those?  (I haven't researched)

**Comment by fosterlynn** [comment]
.... or maybe just use turtle for the diagram and just use the context for the definitions???

**Comment by fosterlynn** [comment]
I have improved it somewhat, so am killing this PR.  New one to follow.

**Comment by ahdinosaur** [comment]
> .... or maybe just use turtle for the diagram and just use the context for the definitions???

or we use turtle for both the diagram and the definitions. the JSON-LD context would then be linking the definitions to how it works in JSON-LD.

**Pull request closed without merging**